bedrock/firefox/templates/firefox/channel/ios.html

```diff
@@ -9,8 +9,6 @@
 {% block page_title %}
 {% if l10n_has_tag('firefox_title_meta_bug1345338') %}
   {{ _('Try New Features in a Pre-Release iOS Browser | Firefox') }}
-{% else %}
-  {{_('Test beta versions of Firefox for iOS via Apple’s TestFlight program and help make our mobile browser for iPhone, iPad and iPod touch even better.')}}
```

Comment:
Looks like this was a mis-paste, there should have been a previous title as the fallback. Just removing this means it would fall back to an empty title. I think we need to restore the old title string but not sure what the l10n impact of that is (if it's already been removed from .lang files)